editor/plugins/script_editor_plugin.cpp

```diff
@@ -3801,6 +3811,23 @@ void ScriptEditor::_on_find_in_files_modified_files(PackedStringArray paths) {
 	_update_modified_scripts_for_external_editor();
 }
 
+void ScriptEditor::_set_zoom_factor(float p_zoom_factor) {
+	if (zoom_factor != p_zoom_factor) {
```

Comment:
This can be early return instead:
```suggestion
	if (zoom_factor == p_zoom_factor) {
		return
	}
```